rainbow-me/browser-extension
# [BX-1294] Fix missing wallets in Send flow

Fixes BX-1294

## What changed (plus any additional context for devs)
- was previously just checking the order of wallets in `useWalletOrderStore()` and returning those. this was only ever updated when the order was changed in the walletSwitcher.
- I updated the send flow to check the existing list of wallets in `getAccounts()` and then compare that to the order of the wallets in `useWalletOrderStore()`
- it then updates the `useWalletOrderStore()` with all the wallets in the correct order and also updated the `useWalletOrderStore()`

## What to test
- see if send suggestions includes all the wallets in your bx aside from the current wallet
- make sure code makes sense

## Screen recordings / screenshots

<video src="https://github.com/rainbow-me/browser-extension/assets/41711440/1d09df63-322f-4255-8be3-dbe94c520328" >
